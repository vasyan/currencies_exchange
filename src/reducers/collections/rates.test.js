import reducer from './rates'

describe('Rates reducer', () => {
  it('should return initial state', () => {
    expect(reducer(undefined, {})).toMatchSnapshot()
  })

  it('should change currency for exchange to', () => {
    expect(
      reducer(undefined, {
        type: 'rates/GET_RATES_SUCCESS',
        payload: { base: 'FOO', rates: { BAR: 42 } }
      })
    ).toMatchSnapshot()
  })
})
