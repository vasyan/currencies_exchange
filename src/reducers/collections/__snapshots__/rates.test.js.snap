// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rates reducer should change currency for exchange to 1`] = `
Object {
  "FOO": Object {
    "BAR": 42,
  },
}
`;

exports[`Rates reducer should return initial state 1`] = `Object {}`;
